src/CharacterList.js

```diff
@@ -57,14 +90,19 @@ class CharacterList extends Component {
         <ul className="nav_items">
           {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} selected={this.state.letter === letter} />)}
         </ul>
-        {!this.state.characters.length && <Loader />}
+        {!this.state.total && <Loader />}
```

Comment:
total always comes back from the request. If the request fails, we have other issues.